doc/nrf/ug_thread_tools.rst

```diff
@@ -196,6 +167,17 @@ However, this approach requires you to download the OpenThread Border Router rep
 To set up and configure the OpenThread Border Router, follow the official `OpenThread Border Router Codelab tutorial`_ on the OpenThread documentation portal.
 Omit the *Build and flash RCP firmware* section, because this section duplicates the steps performed in the previous section.
 
+.. note::
+
+   OTBR is developing every day and it may happen that the newest version is incompatible with the specific version of NCS.
+   To make sure that everything is working correctly, use a specific commit of OTBR that has been tested with the corresponding version of NCS.
+
+   +--------------------------+--------------------+
+   | OpenThread Border Router |        NCS         |
+   +==========================+====================+
+   |         e149a60          |       v1.7         |
```

Comment:
I would prefer to have a simple command instead of the table. I don't think we need to maintain the table with versions for all releases anyway right? Users can switch between documentation of different releases.

So just have a sentence that: when following the CodeLab tutorial, after cloning the repository please ensure to checkout to the compatible commit id.
`git chekout e149a60`